apps/frontend/src/app/5_pages/EarnPage/EarnPage.tsx

```diff
@@ -57,12 +64,27 @@ const EarnPage: FC = () => {
 
   const { account } = useAccount();
   const { value: block } = useBlockNumber();
+  const { price } = useGetRBTCPrice();
+  const [isRecoveryMode, setIsRecoveryMode] = useState(false);
+  const [isUnderCollateralized, setIsUnderCollateralized] = useState(false);
+  const [showUnderCollateralizedError, setShowUnderCollateralizedError] =
```

Comment:
You don't need this variable, you just always set it to whatever value has `isUnderCollateralized`. Or does it have some functionality?

Reply:
Discussed offline, it needs to be there because we want to show the error only after clicking on submit.